yourgov/components/FormMobileFoodVendorPermit/globalFormTasks.ts

```diff
@@ -1,26 +1,11 @@
+// TOOD: Remove entirely? Or keep for easier future-proofing?
```

Comment:
lol `TOOD`. 

>Remove entirely? Or keep for easier future-proofing?

May as well delete since it's not used at all. Certainly once the TODOs are complete.